Tasks/AzureRmWebAppDeploymentV3/operations/AzureAppServiceUtility.ts

```diff
@@ -73,7 +73,8 @@ export class AzureAppServiceUtility {
             var webRequest = new webClient.WebRequest();
             webRequest.method = 'GET';
             webRequest.uri = applicationUrl;
-            var response = await webClient.sendRequest(webRequest);
+            let webRequestOptions = {retriableErrorCodes: [], retriableStatusCodes: [], retryCount: 1, retryIntervalInSeconds: 5};
```

Comment:
As discussed offline, can we have a provision for customer to parse a list of retriable codes if required? Should we add this story to backlog?

Reply:
sure this can be taken to backlog. since this is out of scope for this PR, I am merging it.